lib/msf/core/payload/windows/reverse_http.rb

```diff
@@ -36,7 +36,10 @@ def initialize(*args)
         OptPort.new('PayloadProxyPort', [false, 'An optional proxy server port']),
         OptString.new('PayloadProxyUser', [false, 'An optional proxy server username']),
         OptString.new('PayloadProxyPass', [false, 'An optional proxy server password']),
-        OptEnum.new('PayloadProxyType', [false, 'The type of HTTP proxy (HTTP or SOCKS)', 'HTTP', ['HTTP', 'SOCKS']])
+        OptEnum.new('PayloadProxyType', [false, 'The type of HTTP proxy (HTTP or SOCKS)', 'HTTP', ['HTTP', 'SOCKS']]),
+        OptString.new('HttpHeaderHost', [false, 'An optional value to use for the Host HTTP header']),
```

Comment:
Might be worth moving these options to a mixin that can be included to lib/msf/core/payload/windows/reverse_http.rb, lib/msf/core/payload/android/reverse_http.rb
lib/msf/core/payload/python/reverse_http.rb